nomenclature/definitions/variable/technology/power-plant.yaml

```diff
@@ -230,4 +230,12 @@ Storage Type|Electricity|Energy Storage System:
 
 Stored Energy Inventory|Electricity|Energy Storage System:
    description: State of the stored energy by ESS
-   unit: [MWh, GWh]
\ No newline at end of file
+   unit: [MWh, GWh]
+   
+Energy Charge|Electricity|Energy Storage System:
+   description: Storing energy in the energy storage system unit.
+   unit: [MWh, GWh]
+
+Energy Discharge|Electricity|Energy Storage System:
+   description: Consuming energy from the energy storage system unit.
```

Comment:
This description is not clear - is it consuming energy from the storage into the grid or the other way around?